frameworks/kafka/universe/config.json

```diff
@@ -364,18 +375,44 @@
           "type": "string",
           "default": "9223372036854775807"
         },
+        "log_flush_start_offset_checkpoint_interval_ms": {
+          "title": "log.flush.start.offset.checkpoint.interval.ms",
+          "description": "The frequency with which we update the persistent record of log start offset",
+          "type": "integer",
+          "default": 60000
```

Comment:
For this and other settings where a range is specified in the https://kafka.apache.org/documentation/ documentation. Should we add a `minimum` specifier to the JSON Schema? (Or at least create a ticket to do this?)